node/external/logs/logsRepository.go

```diff
@@ -12,25 +15,25 @@ type logsRepository struct {
 	marshalizer marshal.Marshalizer
 }
 
-func newLogsRepository(args argsNewLogsRepository) *logsRepository {
-	storer := args.StorageService.GetStorer(dataRetriever.TxLogsUnit)
+func newLogsRepository(storageService dataRetriever.StorageService, marshalizer marshal.Marshalizer) *logsRepository {
```

Comment:
could have been renamed to marshaller (we are slowly move away from the "marshalizer" naming)

Reply:
Renamed (but only in package `/logs` for now).